esm-samples/jsapi-vue-cli/package.json

```diff
@@ -1,36 +1,17 @@
 {
-  "name": "vue-cli",
-  "version": "0.1.0",
-  "private": true,
+  "name": "vue-cli-vite",
+  "version": "0.0.0",
   "scripts": {
-    "serve": "vue-cli-service serve --open --port 3001 --host localhost",
-    "build": "vue-cli-service build"
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
   "dependencies": {
     "@arcgis/core": "^4.22.2",
-    "core-js": "^3.19.2",
-    "vue": "^3.2.23"
+    "vue": "^3.2.25"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "~4.5.0",
-    "@vue/cli-plugin-eslint": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
-    "babel-eslint": "^10.1.0",
-    "eslint": "^6.7.2",
-    "eslint-plugin-vue": "^7.0.0"
-  },
-  "eslintConfig": {
-    "root": true,
-    "env": {
-      "node": true
-    },
-    "extends": [
-      "plugin:vue/vue3-essential",
-      "eslint:recommended"
-    ],
-    "parserOptions": {
-      "parser": "babel-eslint"
-    },
-    "rules": {}
+    "@vitejs/plugin-vue": "^2.0.0",
+    "vite": "^2.7.2"
   }
-}
+}
```

Comment:
Set editor to add new line to end of files if possible.

Reply:
Done.